src/pages/rules/Rules.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, Card, CardActions, CardContent, List, ListItem, ListItemText, Typography } from '@mui/material';
```

Comment:
Can remove the console.logs in this file

Reply:
sure